src/ostorlab/agent/schema/target_group_schema.json

```diff
@@ -0,0 +1,182 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Target Group Schema",
+  "description": "Target Group json-specification : A full description of fields of a target group definition yaml file.",
+  "CustomTypes": {
+    "androidStoreType": {
+      "description": "[Optional] - Android store asset.",
+      "type": "object",
+      "properties": {
+        "package_name": {
+          "description": "[Required] - Package name of the app.",
+          "type": "string"
+        }
+      }
+    },
+    "androidApkFileType": {
+      "description": "[Optional] - Android app file asset.",
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "[Required] - Android app file path.",
+          "type": "string"
+        }
+      }
+    },
+    "androidAabFileType": {
+      "description": "[Optional] - Android app file asset.",
+      "type": "object",
+      "properties": {
+        "path": {
+          "description": "[Required] - Android app file path.",
+          "type": "string"
+        }
+      }
```

Comment:
This is missing support for `content_url`, are you intending to use path for both?
If yes, handling this case in the `from_yaml` function is missing.

Reply:
added